src/pages/page/[page].astro

```diff
@@ -17,10 +17,12 @@ const { page } = Astro.props as { page: Page<PostInfo> };
 const pageIndex = page.currentPage;
 
 const SEOTitle = `Post page ${pageIndex}`;
+
+const SEODescription = `Learning programming from magically majestic words. A place to learn about all sorts of programming topics from entry-level concepts to advanced abstractions. Browse all posts on Unicorn Utterances, page ${pageIndex}`;
 ---
 
 <Document>
-	<SEO slot="head" title={SEOTitle} />
+	<SEO slot="head" title={SEOTitle} description={SEODescription} />
```

Comment:
Please remove this commit from this PR; we want PRs to be cleanly separate from one-another